### ISSUE TITLE
Sensitive volumes in `pfRICH_complex.gdml`?

### ISSUE THREAD

**Issue by plexoos** [BNLNPPS/esi-g4ox#42]
I don't see any (optical) volumes with defined EFFICIENCY property in `pfRICH_complex.gdml`. Which ones are supposed to be treated as detectors with hits?

**Comment by ggalgoczi** [comment]
As described in #41 pfRICH_complex.gdml can not be read into Opticks as is, therefore I did not yet add any EFFICIENCY. 

Very likely PhotoDetector will be sensitive but I need to confirm this with pfRICH folks.

**Comment by plexoos** [comment]
Thanks! Could you also confirm with them if the detector is really filled with Nitrogen as the GasVolume volume suggests?

**Comment by ggalgoczi** [comment]
I managed to get Opticks working with the raindrop model, I set the sensitive volume there by assigning to the surface:

`<property name="EFFICIENCY" ref="EFFICIENCY" />`

And adding to the sensitive volume:

`<auxiliary auxtype="SensDet" auxvalue="PhotonDetector" />`

Can I close this issue or should we live open until we get the full detector working?

I will confirm the gas as well with them, sure!

**Comment by plexoos** [comment]
Let's keep it open for now please. I would like to have a better understanding of what needs to be changed in pfRICH_complex.gdml to get Geant hits. I am probably missing something but it sounds like the EFFICIENCY property needs to be set only for Opticks but not for Geant simulation.

**Comment by ggalgoczi** [comment]
That is correct. For Geant4 you define sensitive volume by hand in code. So now you would like to add "sensitivity" for g4ox in the Geant4 context?
